### PR TITLE
azure: replace node build job

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml [hunk 1]
```diff
@@ -18,12 +18,10 @@ postsubmits:
         - --target=[output:stable:startup]
         - --target=[output:stable:sync]
         - --target=e2e-azure
-        - --template=/usr/local/e2e-azure
+        - --template=/usr/local/cluster-launch-e2e-azure.yaml
```
Comment (mjudeikis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: We used this as a folder. Previously it was an issue with referring as a file. Need to test with mkpj

**Comment by mjudeikis** [comment]
@jim-minter:
to test job: branch-ci-openshift-openshift-azure-master-e2e-azure
check out branch you testing in release repo (and chose PR with no changes)
```
hack/pj_env.py \
    openshift/openshift-azure master 1158 'Pull Request Author' \
    hack/mkpjpod.sh branch-ci-openshift-openshift-azure-master-e2e-azure \
    | oc -n ci-stg create -f -
```

File all this from Cli where you have CI Cluster CLI:
```
mjudeiki /home/mjudeiki/go/src/github.com/openshift/release git:azure-node-native 
→ hack/pj_env.py \
>     openshift/openshift-azure master 1158 'Pull Request Author' \
>     hack/mkpjpod.sh branch-ci-openshift-openshift-azure-master-e2e-azure \
>     | oc -n ci-stg create -f -
time="2019-03-12T15:46:11Z" level=warning msg="No BuildID found in ProwJob status or given with --build-id, GCS interaction will be poor." 
pod "f5a095f7-44dd-11e9-806c-0242ac110003" created

mjudeiki /home/mjudeiki/go/src/github.com/openshift/release git:azure-node-native 
→ oc logs -f f5a095f7-44dd-11e9-806c-0242ac110003 -c ci0stg
Error from server (Forbidden): pods "f5a095f7-44dd-11e9-806c-0242ac110003" is forbidden: User "mjudeikis" cannot get pods in the namespace "ci-op-wgp0vt5d": no RBAC policy matched

mjudeiki /home/mjudeiki/go/src/github.com/openshift/release git:azure-node-native 
→ oc logs -f f5a095f7-44dd-11e9-806c-0242ac110003 -c ci-stg
Error from server (Forbidden): pods "f5a095f7-44dd-11e9-806c-0242ac110003" is forbidden: User "mjudeikis" cannot get pods in the namespace "ci-op-wgp0vt5d": no RBAC policy matched

mjudeiki /home/mjudeiki/go/src/github.com/openshift/release git:azure-node-native 
→ oc logs -f f5a095f7-44dd-11e9-806c-0242ac110003 -n ci-stg
Error from server (BadRequest): a container name must be specified for pod f5a095f7-44dd-11e9-806c-0242ac110003, choose one of: [test sidecar] or one of the init containers: [clonerefs initupload place-tools]

mjudeiki /home/mjudeiki/go/src/github.com/openshift/release git:azure-node-native 
→ oc logs -f f5a095f7-44dd-11e9-806c-0242ac110003 -n ci-stg -c test
2019/03/12 15:47:15 Resolved source https://github.com/openshift/openshift-azure to master@edc36fc8
2019/03/12 15:47:15 Resolved https://api.ci.openshift.org/azure/test-base:latest to sha256:84aea397867d2144ab35fb4850dda0cd67e7484073971a47da0c77ddc881ee22
2019/03/12 15:47:15 Resolved https://api.ci.openshift.org/azure/plugin-base:latest to sha256:d31b9c3d4868910e1c4acdd9d38390f7467825664e6b6c2f4ca3410fb8e50dfb
2019/03/12 15:47:15 Using namespace ci-op-zbxfcc0z
2019/03/12 15:47:15 Ran for 0s
error: could not build execution graph: the following names were not found in the config or were duplicates: e2e-azure (from [input:root], src, bin, [input:base], sync, [output:stable:sync], etcdbackup, [output:stable:etcdbackup], startup, [output:stable:startup], azure-controllers, [output:stable:azure-controllers], metricsbridge, [output:stable:metricsbridge], e2e-tests, [output:stable:e2e-tests], verify, unit, e2e-upgrade-v2.4, e2e-upgrade-v2.5, [release:latest], [release:initial], [release-inputs], cluster-launch-e2e-azure, [images])
{"component":"entrypoint","error":"wrapped process failed: exit status 1","level":"error","msg":"Error executing test process","time":"2019-03-12T15:47:15Z"}
```
Gives error on this, and I suspect this is folder and `--target` rename issue. 
`error: could not build execution graph: the following names were not found in the config or were duplicates: e2e-azure `

and e2e-azure as a target comes from:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml#L79
AND template name (which is different now)

So target should be named `cluster-launch-e2e-azure` or rolled back.

**Comment by jim-minter** [comment]
/hold
refactoring

**Comment by jim-minter** [comment]
/hold cancel
still WIP

**Comment by jim-minter** [comment]
@mjudeikis xref https://github.com/openshift/openshift-azure/pull/1274

**Comment by jim-minter** [comment]
@mjudeikis I've tested as best I can with mkpg - now wdyt?

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v2-presubmits.yaml [hunk 2]
```diff
@@ -20,22 +20,18 @@ presubmits:
...
-          value: azure
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
               key: openshift-openshift-azure-release-v2.yaml
               name: ci-operator-misc-configs
         - name: DEPLOY_VERSION
           value: v3.11
-        - name: JOB_NAME_SAFE
-          value: e2e-azure
```
Comment (mjudeikis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v2-presubmits.yaml]: I think we need these:
```
  JOB_NAME_SAFE
    The job name in a form safe for use as a Kubernetes resource name.
```

**Comment by mjudeikis** [comment]
raised https://github.com/openshift/release/pull/3143 just so I can debug and rebase

**Pull request merged**